test/service_stubs/heroku_api_stub.rb

```diff
@@ -164,11 +164,12 @@ def two_factor?
     })
   end
 
-  post "/invitation2/save" do
+  patch "/invitations/:token" do
     MultiJson.encode({
-      email: "kerry@heroku.com",
-      signup_source: {
-        redirect_uri: "https://dashboard.heroku.com"
+      created_at: Time.now,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.